pkg/controller/v1beta1/inferenceservice/reconcilers/ingress/rawingress_reconciler.go

```diff
@@ -0,0 +1,176 @@
+	v1beta1api "github.com/kubeflow/kfserving/pkg/apis/serving/v1beta1"
+	"github.com/kubeflow/kfserving/pkg/constants"
+	corev1 "k8s.io/api/core/v1"
+	netv1 "k8s.io/api/networking/v1"
+	"k8s.io/apimachinery/pkg/api/equality"
+	apierr "k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/types"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+//NativeReconciler is the struct of Native K8S Object
+type RawIngressReconciler struct {
+	client          client.Client
+	scheme          *runtime.Scheme
+	Ingress         *netv1.Ingress
+	componentExt    *v1beta1.ComponentExtensionSpec
+	componentStatus v1beta1.ComponentStatusSpec
+}
+
+func NewRawIngressReconciler(client client.Client,
+	scheme *runtime.Scheme,
+	componentMeta metav1.ObjectMeta,
+	componentExt *v1beta1.ComponentExtensionSpec,
+	podSpec *corev1.PodSpec,
+	componentStatus v1beta1.ComponentStatusSpec,
+	componentType v1beta1.ComponentType) *RawIngressReconciler {
+	return &RawIngressReconciler{
+		client:          client,
+		scheme:          scheme,
+		Ingress:         createRawIngress(client, componentMeta, componentExt, podSpec, componentStatus, componentType),
+		componentExt:    componentExt,
+		componentStatus: componentStatus,
+	}
+}
+
+//GenerateIngressHost return the config domain in configmap.IngressDomain
+func GenerateIngressHost(client client.Client, metadata metav1.ObjectMeta) string {
+	ingressConfig, err := v1beta1api.NewIngressConfig(client)
+	if err != nil {
+		log.Error(err, "GenerateIngressHost")
+		if ingressConfig == nil {
+			ingressConfig = &v1beta1api.IngressConfig{}
+		}
+		if ingressConfig.IngressDomain == "" {
+			ingressConfig.IngressDomain = "example.com"
+		}
+	}
+	return metadata.Name + "-" + metadata.Namespace + "." + ingressConfig.IngressDomain
+}
+
+func createRawIngress(client client.Client, componentMeta metav1.ObjectMeta,
+	componentExt *v1beta1.ComponentExtensionSpec,
+	podSpec *corev1.PodSpec,
+	componentStatus v1beta1.ComponentStatusSpec,
+	componentType v1beta1.ComponentType) *netv1.Ingress {
+	ingressHost := GenerateIngressHost(client, componentMeta)
+	pathType := netv1.PathType(netv1.PathTypeImplementationSpecific)
+	ingress := &netv1.Ingress{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      componentMeta.Name,
+			Namespace: componentMeta.Namespace,
+		},
+		Spec: netv1.IngressSpec{
+			Rules: []netv1.IngressRule{
+				{
+					Host: ingressHost,
+					IngressRuleValue: netv1.IngressRuleValue{
+						HTTP: &netv1.HTTPIngressRuleValue{
+							Paths: []netv1.HTTPIngressPath{
+								{
+									Path:     "/" + getIngressPath(componentType),
+									PathType: &pathType,
+									Backend: netv1.IngressBackend{
+										Service: &netv1.IngressServiceBackend{
+											Name: componentMeta.Name,
+											Port: netv1.ServiceBackendPort{
+												Number: podSpec.Containers[0].Ports[0].ContainerPort,
```

Comment:
If service port is set to 80 here can use the same.

Reply:
sorry, I can not get this meaning